htdocs/admin/stock.php

```diff
@@ -335,7 +343,7 @@
 print '<td class="right">';
 if (isModEnabled("supplier_order") || isModEnabled("supplier_invoice")) {
 	if ($conf->use_javascript_ajax) {
-		if ($disabled) {
+		if (in_array('BILL', $disableStockCalculateOn)) {
```

Comment:
I think you want to do here ?
`if (in_array('SUPPLIER_BILL', $disableStockCalculateOn)) {`